plugins/available/base.plugin.bash

```diff
@@ -183,3 +183,16 @@ if ! _command_exists del; then
 		mkdir -p /tmp/.trash && mv "$@" /tmp/.trash
 	}
 fi
+
+# replace multiple file extensions at once
+function rex() {
+	about 'mass replace of the extension of multiple files'
+ 	param '1: extension to replace'
```

Comment:
The indentation is mixed spaces and tabs, please align all to just tabs. Thanks!